src/software/RigCalibration/main_cameraCalibration.cpp

```diff
@@ -1,210 +1,234 @@
-                rvecs, tvecs, cameraMatrix, distCoeffs, reprojErrs);
+  startrC = std::clock();
+  
+  totalAvgErr = computeReprojectionErrors(objectPoints, imagePoints,
+                                          rvecs, tvecs, cameraMatrix, distCoeffs, reprojErrs);
+  durationrC = ( std::clock() - startrC ) / (double) CLOCKS_PER_SEC;
+  std::cout << "  computeReprojectionErrors duration: "<< durationrC << std::endl;
 
-    return ok;
+  return ok;
 }
 
-static void saveCameraParamsToPlainTxt(const string &filename,
-                                       const Size &imageSize,
-                                       const Mat& cameraMatrix,
-                                       const Mat& distCoeffs)
+static void saveCameraParamsToPlainTxt(const std::string &filename,
+                                       const cv::Size &imageSize,
+                                       const cv::Mat& cameraMatrix,
+                                       const cv::Mat& distCoeffs)
 {
   std::ofstream fs(filename, std::ios::out);
-  if(!fs.is_open())
+  if (!fs.is_open())
   {
     std::cerr << "Unable to create the calibration file " << filename << std::endl;
-    throw std::invalid_argument("Unable to create the calibration file "+filename);
+    throw std::invalid_argument("Unable to create the calibration file " + filename);
   }
-  
+
   // the structure of the file is
-// int #image width
-// int #image height
-// double #focal
-// double #ppx principal point x-coord
-// double #ppy principal point y-coord
-// double #k0
-// double #k1
-// double #k2
+  // int #image width
+  // int #image height
+  // double #focal
+  // double #ppx principal point x-coord
+  // double #ppy principal point y-coord
+  // double #k0
+  // double #k1
+  // double #k2
   fs << imageSize.width << std::endl;
   fs << imageSize.height << std::endl;
-  if(cameraMatrix.type() == DataType<double>::type)
+  if (cameraMatrix.type() == cv::DataType<double>::type)
   {
-    fs << (cameraMatrix.at<double>(0,0) + cameraMatrix.at<double>(1,1))/2 << std::endl;
-    fs << cameraMatrix.at<double>(0,2) << std::endl;
-    fs << cameraMatrix.at<double>(1,2) << std::endl;
+    fs << (cameraMatrix.at<double>(0, 0) + cameraMatrix.at<double>(1, 1)) / 2 << std::endl;
+    fs << cameraMatrix.at<double>(0, 2) << std::endl;
+    fs << cameraMatrix.at<double>(1, 2) << std::endl;
   }
   else
   {
-    fs << (cameraMatrix.at<float>(0,0) + cameraMatrix.at<float>(1,1))/2 << std::endl;
-    fs << cameraMatrix.at<float>(0,2) << std::endl;
-    fs << cameraMatrix.at<float>(1,2) << std::endl;
+    fs << (cameraMatrix.at<float>(0, 0) + cameraMatrix.at<float>(1, 1)) / 2 << std::endl;
+    fs << cameraMatrix.at<float>(0, 2) << std::endl;
+    fs << cameraMatrix.at<float>(1, 2) << std::endl;
   }
-  if(distCoeffs.type() == DataType<double>::type)
+  if (distCoeffs.type() == cv::DataType<double>::type)
   {
     fs << distCoeffs.at<double>(0) << std::endl;
     fs << distCoeffs.at<double>(1) << std::endl;
```

Comment:
@QGimac now we know where that bug inside export came from, it's my fault... line 235 (and 241) should read
```c++
fs << distCoeffs.at<double>(1) << std::endl;
```